build/root/BUILD.root

```diff
@@ -80,3 +80,16 @@ genrule(
     cmd = "grep ^STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt | awk '{print $$2}' >$@",
     stamp = 1,
 )
+
+go_library(
+    name = "go_default_library",
+    srcs = ["test.go"],
```

Comment:
Ops, I have a temp test.go in the root directory and it makes this unrelated code gen. Will update code gen